src/Inkluzitron/Data/Entities/Badge.cs

```diff
@@ -0,0 +1,23 @@
+﻿using ImageMagick;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Inkluzitron.Data.Entities
+{
+    public class Badge
+    {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.None)]
+        public string Id { get; set; }
+
+        [Required]
+        public string Name { get; set; }
+
+        public string Description { get; set; }
+
+        public MagickImage Image { get; set; }
```

Comment:
Tady by, myslím, mělo být [Required]. Buď to, nebo v BotDatabaseContext upravit tu konverzi, aby neshořela, když bude Image/from null.

Reply:
Je required